src/test/setup.js

```diff
@@ -6,12 +6,25 @@
 // Importing this here makes it work everywhere.
 import '@testing-library/jest-dom';
 
+// This installs jest matchers as a side effect as well.
+import fetchMock from 'fetch-mock-jest';
+import { Headers, Request, Response } from 'node-fetch';
+
 jest.mock('../utils/worker-factory');
 import * as WorkerFactory from '../utils/worker-factory';
 import { autoMockResizeObserver } from './fixtures/mocks/resize-observer';
 
 autoMockResizeObserver();
 
+beforeEach(function () {
+  // Install fetch and fetch-related objects globally.
+  // Using the sandbox ensures that parallel tests run properly.
+  global.fetch = fetchMock.sandbox();
```

Comment:
When moving to node v18 and `fetch` will be available globally in node, we might need to revisit the approach.